components/Header/Header.js

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import dynamic from "next/dynamic";
+
+const StyledHeader = styled.header`
+  text-align: center;
+  margin-bottom: 0rem;
+`;
+
+const BackLinkSvg = styled.svg`
+  position: fixed;
+  left: 5px;
+  top: 5px;
+  padding: 8px;
+  z-index: 3;
+  border: 2px solid #3498db;
+  border-radius: 8px;
+  background-color: #ffffff;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+  transition: background-color 0.3s ease, transform 0.3s ease;
+
+  &:hover {
+    background-color: #f0f8ff;
+    transform: scale(1.1);
+  }
+
+  &:active {
+    transform: scale(1.05);
+  }
+`;
+
+const HiddenAppName = styled.h1`
+  display: none;
+`;
+
+export default function Header() {
+  const router = useRouter();
+  const headerContent = getHeaderContent(router.pathname);
+
+  function getHeaderContent(pathname) {
+    switch (pathname) {
+      case "/exercises/[id]":
+        return (
+          <>
+            <Image src="/logo.png" alt="Logo" width={90} height={90} />
+            <Link href={`/`}>
+              <BackLinkSvg
+                xmlns="http://www.w3.org/2000/svg"
+                width="50"
+                height="50"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="#3498db"
+                strokeWidth="3"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              >
+                <polyline points="15 18 9 12 15 6" />
+              </BackLinkSvg>
```

Comment:
If you don't want all that info inside the svg tag you can easily import the svg like this ```import BackLinkSvg from "@/public/icons/BackLinkSvg.svg";``` and then it only looks like this in your code ```<BackLinkSvg />```

Reply:
in the team we decided to keep it like our code and just collapse the Backlink in the code.